README.md

```diff
@@ -0,0 +1,52 @@
+# Difflint #
+
+Do you have a codebase with inconsistent code style, and you want to
```

Comment:
Phrasing this as a question gives it a marketing vibe that feels out of place in a README. Consider rephrasing as a statement.